examples/features/orca/client/main.go

```diff
@@ -16,7 +16,7 @@
  *
  */
 
-// Binary client is an example client.
+// Binary client demonstrates how to handle ORCA metrics for load reporting.
```

Comment:
Again, the crux of this example is the custom LB policy that uses ORCA per-call and out-of-band metrics.

Reply:
Done